### PR TITLE
Updated component to accept 'variant' prop; created second story to s…

### PR DESCRIPTION
**Pull request by Carlo-Piantini**
…how 'fullWidth' and 'scrollable' variants

### CODE REVIEW

File: src/lib/tabs/Tabs.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { Tabs as MuiTabs, Tab } from '@material-ui/core';
 
 interface TabPanelProps {
   children?: React.ReactNode;
+  className: any,
```
Comment (jaruesink) [inline src/lib/tabs/Tabs.tsx:8]: children and className shouldn't be needed here.

`React.FC<TabPanelProps>` React.FC stands for React Functional Component and adds all React standard props already.

File: src/lib/tabs/Tabs.tsx [hunk 2]
```diff
@@ -70,9 +71,9 @@ export const Tabs: React.FC<{ ariaLabel?: string; tabs: { label: string; render:
   };
 
   return (
-      <div className={classes.root}>
-      <AppBar className={classes.appBar} position="static">
-        <MuiTabs classes={{indicator: classes.indicator}} value={value} onChange={handleChange} aria-label={ariaLabel}>
+    <div className={classes.root}>
+      <AppBar className={classes.appBar} color='default' position='static'>
```
Comment (jaruesink) [inline src/lib/tabs/Tabs.tsx:75]: is color default needed? seems like that would be assumed

File: src/lib/tabs/Tabs.tsx (hunk 1 above)
Comment (Carlo-Piantini) [reply src/lib/tabs/Tabs.tsx:8]: This way throwing a TS error without having it defined inside of the interface, that error was the reason the first build for `0.3.0` failed. Are you sure it isn't necessary?

File: src/lib/tabs/Tabs.tsx (hunk 2 above)
Comment (Carlo-Piantini) [reply src/lib/tabs/Tabs.tsx:75]: Lol yup! The default of `color` is actually `primary`. Checkout the API: https://material-ui.com/api/app-bar/ 

Without it set to default, the color of the arrows is white and they're not visible to user when they're active.

File: src/lib/tabs/Tabs.tsx (hunk 1 above)
Comment (jaruesink) [reply src/lib/tabs/Tabs.tsx:8]: I'll check her out on my end

File: src/lib/tabs/Tabs.tsx (hunk 2 above)
Comment (jaruesink) [reply src/lib/tabs/Tabs.tsx:75]: haha, that's kinda funny, default is not default

File: src/lib/tabs/Tabs.tsx (hunk 1 above)
Comment (Carlo-Piantini) [reply src/lib/tabs/Tabs.tsx:8]: Just checked, here's the error being thrown by TS:
```[!] (plugin typescript) Error: @rollup/plugin-typescript TS2322: Type '{ children: ReactNode; className: string; value: number; index: number; }' is not assignable to type 'IntrinsicAttributes & TabPanelProps & { children?: ReactNode; }'.
  Property 'className' does not exist on type 'IntrinsicAttributes & TabPanelProps & { children?: ReactNode; }'.```

For whatever reason, it looks like the `<TabPanel>` component doesn't have access to `className` without it being defined in the interface.

Comment (jaruesink) [reply src/lib/tabs/Tabs.tsx:8]: oh className was needed, but not children. I'm actually getting rid of the need for any of these styles I think.

**Review by jaruesink — APPROVED** [review]

**Pull request merged**